src/components/molecules/HeaderDesktop/HeaderDesktop.tsx

```diff
@@ -0,0 +1,10 @@
+const HeaderDesktop = () => (
+  <nav className="flex justify-end px-48 py-12 bg-orange200 w-full">
+    <ul className="flex list-none p-0 text-white">
+      <li className="mr-4 cursor-pointer"><a href="/about">O stronie</a></li>
+      <li className="cursor-pointer"><a href="/contact">Kontakt</a></li>
+    </ul>
+  </nav>
```

Comment:
`ul` and `li` are redundant. You could use `a` directly.
You could also pass links as a prop without hardcoding them.

```suggestion
  <nav className="flex gap-4 justify-end px-48 py-12 bg-orange200 w-full  text-white">
      {links.map(({ href, label }) => (
         <a className="cursor-pointer" href={href} key={href + label}>{label}</a>
      ))}
  </nav>
```